### PR TITLE
Translate to Portuguese: examples/jsf-managedBean-and-ejb

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira -> https://issues.apache.org/jira/browse/TOMEE-2823

### CODE REVIEW

File: examples/jsf-managedBean-and-ejb/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+[source,java]
+----
+       <h:outputText value='Enter first number'/>
+       <h:inputText value='#{calculatorBean.x}'/>
+       <h:outputText value='Enter second number'/>
+       <h:inputText value='#{calculatorBean.y}'/>
+       <h:commandButton action="#{calculatorBean.add}" value="Add"/>
+----
+
+Observe como usamos o bean aqui. Por padrão, é a classe simples
```
Comment (daniel-augusto) [inline examples/jsf-managedBean-and-ejb/README_pt.adoc:57]:
```suggestion
Observe como usamos o bean aqui. Por padrão, o nome do Managed Bean é o nome da classe.
```

File: examples/jsf-managedBean-and-ejb/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+nome do Managed Bean.
```
Comment (daniel-augusto) [inline examples/jsf-managedBean-and-ejb/README_pt.adoc:58]: Remove this line.

File: examples/jsf-managedBean-and-ejb/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+Quando uma requisição é recebida, o bean é instanciado e colocado no escopo apropriado. Por padrão, o bean é colocado no escopo request.
+
+[source,java]
+----
+        <h:inputText value='#{calculatorBean.x}'/>
+----
+
+Aqui, o método `getX()` de calculatorBean é chamado e o valor resultante é exibido. `x` sendo um duplo, com razão, devemos ver 0,0 exibidos.
```
Comment (daniel-augusto) [inline examples/jsf-managedBean-and-ejb/README_pt.adoc:67]:
```suggestion
Aqui, o método `getX()` de calculatorBean é chamado e o valor resultante é exibido. `x` sendo um double, corretamente, devemos ver 0.0 exibidos.
```

File: examples/jsf-managedBean-and-ejb/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+Nesse caso, `CalculatorBean#add()` é chamado.
+
+`Calculator#add()` delega o trabalho para o ejb, obtém o resultado,
+armazena e depois instrui qual visualização deve ser renderizada.
+
+Você está certo. O valor de retorno `success` é verificado nas regras de navegação faces-config e a respectiva página é renderizada.
+
+No nosso caso, a página `result.xhtml` é renderizada.
+
+A solicitação no escopo `calculatorBean` está disponível aqui e usamos EL para exibir os valores.
```
Comment (daniel-augusto) [inline examples/jsf-managedBean-and-ejb/README_pt.adoc:80]:
```suggestion
O bean com escopo de request `calculatorBean` está disponível aqui e usamos EL para exibir os valores.
```

**Comment by Daniel-Dos** [comment]
great @daniel-augusto  thanks .

**Pull request merged**